ouroboros-consensus-cardano/changelog.d/20240823_131440_damian.nadales_324_simplify_protver_arguments.md

```diff
@@ -0,0 +1,4 @@
+### Breaking
+
+- ProtocolParams (CardanoBlock c) was monomorphized to `CardanoProtocolParams`
+- Remove `cardanoProtocolParamsPerEra` from `CardanoProtocolParams` in favour of a single `cardanoProtocolVersion`.
```

Comment:
I think this could include a super-brief rationale, ie that this intentionally removes flexibility in favor of a simpler interface as the current interface has caused lots of confusion and mistakes in the past.